patches/net/minecraft/client/gui/screens/options/controls/KeyBindsScreen.java.patch

```diff
@@ -40,10 +40,13 @@
 +        }
 +    }
 +
-+    // Neo: This method is override to more easily handle modifier keys
++    // Neo: This method is overridden to more easily handle modifier keys
 +    @Override
 +    public boolean keyReleased(int p_94715_, int p_94716_, int p_94717_) {
-+        if (this.selectedKey != null) {
++        // We ignore events from keys with the scan code 63 as they're emitted
++        // (only as RELEASE, not PRESS) by Mac systems to indicate that "Fn" is being pressed
++        // See https://github.com/neoforged/NeoForge/issues/1683
++        if (this.selectedKey != null && (net.minecraft.Util.getPlatform() != net.minecraft.Util.OS.OSX || p_94716_ != 63)) {
```

Comment:
```suggestion
+        if (this.selectedKey != null && (!net.minecraft.client.Minecraft.ON_OSX || p_94716_ != 63)) {
```

Reply:
Great, so that's the 3rd way I've found to check whether you're on a Mac in Minecraft code